src/view_slur.cpp

```diff
@@ -58,7 +58,7 @@ void View::DrawSlur(DeviceContext *dc, Slur *slur, int x1, int x2, Staff *staff,
     if (graphic)
         dc->ResumeGraphic(graphic, graphic->GetUuid());
     else
-        dc->StartGraphic(slur, "spanning-slur", "");
+        dc->StartGraphic(slur, slur->GetSpanningClasses(), slur->GetUuid());
```

Comment:
Any reasons why passing the third parameter for slurs and not for other control events?

Reply:
That's of course wrong. I corrected that.